EKF/estimator_base.cpp

```diff
@@ -170,7 +171,7 @@ void EstimatorBase::setGpsData(uint64_t time_usec, struct gps_message *gps)
 		gps_sample_new.time_us -= FILTER_UPDATE_PERRIOD_MS * 1000 / 2;
 		_time_last_gps = time_usec;
 
-		_last_valid_gps_time_us = hrt_absolute_time();
+        _last_valid_gps_time_us = _time_last_imu;
```

Comment:
@priseborough Do we really need this? We already have _time_last_gps above. The two variables have exactly the same value I believe.